worlds/oot_soh/Options.py

```diff
@@ -749,6 +749,13 @@ class SkeletonKey(Toggle):
     display_name = "Skeleton Key"
 
 
+class SlingbowBreakBehives(Toggle):
+    """
+    Both of the Slingshot or Fairy Bow can be used to break beehives if beehives are shuffled.
+    """
```

Comment:
```suggestion
    Allows Slingshot and Bow to break beehives when Beehive Shuffle is turned on.
    """
```

As per ship description